tools/cldr-code/src/main/java/org/unicode/cldr/test/ConsoleCheckCLDR.java

```diff
@@ -358,9 +359,9 @@ public static void main(String[] args) throws IOException {
         if (options[VOTE_RESOLVE].doesOccur) {
             resolveVotesDirectory = CldrUtility.checkValidFile(CLDRPaths.BASE_DIRECTORY + "incoming/vetted/votes/",
                 true, null);
-            VoteResolver.setVoterToInfo(CldrUtility.checkValidFile(CLDRPaths.BASE_DIRECTORY
+            voterInfoList.setVoterToInfo(CldrUtility.checkValidFile(CLDRPaths.BASE_DIRECTORY
                 + "incoming/vetted/usersa/usersa.xml", false, null));
```

Comment:
just curious -- where does usersa.xml come from and what does it look like?

Reply:
@btangmu  I haven't been able to find the generation side. Did it get removed? OutputFileManager used to produce it.

Reply:
https://github.com/unicode-org/cldr-aux/blob/master/voting/36/usersa/usersa.xml

Reply:
Removed as dead code in https://github.com/unicode-org/cldr/pull/864

Reply:
Yes, and its cause of death has been lost in the mists of time.

Reply:
I vaguely remember we discussed it in a meeting and the consensus was that the db backups were sufficient for a history of voting, so we didn't need an xml version. At that time I wasn't aware that console check could read the xml. Now I see we have VOTE_RESOLVE and VoteResolver.setVoterToInfo... setVoterToInfo(String fileName) is only called by console check, so that's also dead unless/until usersa.xml gets resurrected. There's also setVoterToInfo(Map<Integer, VoterInfo> testVoterToInfo) which still has callers...